client/components/plans/plan-actions/index.jsx

```diff
@@ -49,11 +49,12 @@ module.exports = React.createClass( {
 	},
 
 	getAtlasButtons: function() {
```

Comment:
We can probably just remove these altogether, since we shouldn't be using Atlas at all. I'll create a PR for it...